### PR TITLE
Update Proxy SHA to latest (release-1.0).

### PR DESCRIPTION
**Pull request by PiotrSikora**

### CODE REVIEW

**Comment by PiotrSikora** [comment]
Duplicate of #7108 with `go-control-plane` updates to see if they fix the stuck tests.

**Comment by kyessenov** [comment]
Hm, I'd be surprised if go-control-plane update affected the tests

**Comment by PiotrSikora** [comment]
@rshriram this is `HEAD~1`, which didn't require other changes.

**Comment by rshriram** [comment]
Same failure.

**Comment by kyessenov** [comment]
Can we rollback envoy update to get mine and @bianpengyuan 's changes to proxy? We're approaching the end of the cycle...

**Comment by PiotrSikora** [comment]
@kyessenov you mean in [istio/proxy](https://github.com/istio/proxy)? Could you do 2 commits (revert and roll-forward) and merge the reverted one? We need to get this one in sooner rather than later...

**Comment by kyessenov** [comment]
@rshriram can you revert Piotr's proxy PR to update envoy so we can move forward with other changes. Either this, or please fix this PR.

**Comment by rshriram** [comment]
Sure. please send a PR to proxy.

File: tests/testdata/multicluster/envoy_local_v2.yaml [hunk 1]
```diff
@@ -7,11 +7,15 @@ dynamic_resources:
...
-      cluster_names: [xds_cluster]
+      grpc_services:
+        envoy_grpc":
+          cluster_name: "xds_cluster"
   cds_config:
     api_config_source:
       api_type: GRPC
-      cluster_names: [xds_cluster]
+      grpc_services:
+        envoy_grpc":
```
Comment (ijsnellf) [inline tests/testdata/multicluster/envoy_local_v2.yaml]: random `"`

File: tests/testdata/multicluster/envoy_local_v2.yaml [hunk 2]
```diff
@@ -7,11 +7,15 @@ dynamic_resources:
   lds_config:
     api_config_source:
       api_type: GRPC
-      cluster_names: [xds_cluster]
+      grpc_services:
+        envoy_grpc":
```
Comment (ijsnellf) [inline tests/testdata/multicluster/envoy_local_v2.yaml]: random `"`

Comment (PiotrSikora) [reply tests/testdata/multicluster/envoy_local_v2.yaml]: Good catch! Unfortunately, that wasn't the issue.

File: tests/istio.mk [hunk 3]
```diff
@@ -176,15 +176,17 @@ test/local/e2e_galley: out_dir istioctl generate_yaml
 	${CAPTURE_LOG}
 
 # v1alpha3+envoyv2 test without MTLS
-test/local/noauth/e2e_pilotv2: out_dir generate_yaml
+test/local/noauth/e2e_pilotv2: out_dir
```
Comment (kyessenov) [inline tests/istio.mk]: cc @hklai : is this scoped to circle only? We do want to test the whole package on prow.

Comment (rshriram) [reply tests/istio.mk]: this is just for temporary testing

File: tests/e2e/framework/kubernetes.go [hunk 4]
```diff
@@ -324,13 +323,6 @@ func (k *KubeInfo) Setup() error {
 				log.Error("Failed to deploy Istio.")
 				return err
 			}
-
```
Comment (andraxylia) [inline tests/e2e/framework/kubernetes.go]: why is this being removed?

Comment (andraxylia) [reply tests/e2e/framework/kubernetes.go]: nvm, found it.

**Comment by rshriram** [comment]
closing.. memory leaks found in proxy.. please fix and open again

**Pull request closed without merging**